images/scripts/services.setup

```diff
@@ -35,6 +35,11 @@ EOF
 chmod 755 /root/run-freeipa
 /root/run-freeipa
 
+# HACK: Podman fails to relabel symlinks properly, accidentally relabelling their targets instead
+# https://github.com/opencontainers/selinux/pull/173
+# Disable relabelling for subsequent runs.
+sed -ie 's/data:Z/data/' /root/run-freeipa
```

Comment:
-ie is actually creating a backup. Would you consider changing it to -i?

Reply:
Thanks for the catch.  Updated.